airflow/utils/dag_processing.py

```diff
@@ -539,7 +541,9 @@ def helper():
             # e.g. RotatingFileHandler. And it can cause connection corruption if we
             # do not recreate the SQLA connection pool.
             os.environ['CONFIG_PROCESSOR_MANAGER_LOGGER'] = 'True'
-            reload_module(airflow.config_templates.airflow_local_settings)
+            # Replicating the behavior of how logging module was loaded
+            # in logging_config.py
+            reload_module(import_module(logging_class_path.rsplit('.', 1)[0]))
```

Comment:
Do we need `[0]` here ?? `.rsplit('.',1)` would just have one item right??

Reply:
It will actually [return a list as split](https://docs.python.org/2/library/string.html#string.rsplit). If not it will fail the newly added unit test :D

Reply:
Oh yes, my bad 😄